Add my name to the contributors list

kylemcdermed opened the pull request:
Before submitting this pull request, check the changes to see it's only the changes you made intentionally
If there are changes to other lines you didn't make deliberately, it's possible that your IDE made the changes with a utility like prettier.
Next time, make sure that you only add your changes by using `git add -p` and rather than `git add Contributors.md`

If you're doing something in the checklist below, put an `x` inside `[ ]` so that `- [ ]` becomes `- [x]`

- [x] I had fun going through this tutorial (ノ^o^)ノ and learned on the way ٩(＾◡＾)۶
- [ ] There are some things I'd like to improve in this tutorial. I have written them below.
- [ ] There were steps where I had errors while following this tutorial. I have written them below.

kylemcdermed committed:
Add my name to the contributors list

Roshanjossey commented:
Hi, I appreciate you cleaning up the format but this results in multiple merge conflicts and prevents me from merging. Could you please submit another pull request by adding your name only?

If you really wanted to make these changes, please read [design decisions](https://github.com/firstcontributions/first-contributions/issues/35892)

If you didn't make changes to other lines deliberately, it's possible that your IDE made the changes with a utility like prettier. Next time, make sure that you only add your changes by using `git add -p` and rather than `git add Contributors.md`

Feel free to ask me any questions if you need further clarifications :)

Cheers

kylemcdermed commented:
Hi! I’ve updated the pull request per your request. I’ve added only my name to the Contributors.md file, and the formatting changes have been removed. Let me know if there’s anything else I can do. Thanks for your help!
